### DISCUSSION TITLE
Does G-helper supports TUF A-15?

### DISCUSSION THREAD

**Discussion by Alex20255** [seerge/g-helper#3382] [Q&A]
I just bought a new laptop and downloaded G-helper but I don't have any mode selection from GPU. I only have stop GPU application button. 
I found some replies said G-helper may not support old models but mine should be new(FA507NVR)

Sorry for my capture in Chinese
![螢幕擷取畫面 2024-11-16 183725](https://github.com/user-attachments/assets/47ed8eb4-85be-48ad-926f-4d8c5436208f)

**Comment by seerge** [comment]
@Alex20255 yes, it does support any modern asus laptops. If you never had GPU modes in Armory - you won’t have them here, as your device may not support them
